ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/common/utils/LocaleUtilsTest.java

```diff
@@ -0,0 +1,44 @@
+package com.kayako.sdk.android.k5.common.utils;
+
+import com.kayako.sdk.helpcenter.locale.Locale;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class LocaleUtilsTest {
+    private static final String LOCALE_SPAIN = "es-ES";
+    private static Locale helpCenterLocale;
+
+    @Before
+    public void setUp() {
+        helpCenterLocale = new Locale();
+        helpCenterLocale.setLocale(LOCALE_SPAIN);
+    }
+
+    @Test
+    public void givenLocalWhenGetLocaleThenReturnUtilLocale() {
```

Comment:
⚠️  This snippet of code contains typos. Spelling errors make reading and understanding the source code harder - mild errors cause the reader to stumble for a fraction of a second, many of them can cause considerable delays; really bad errors can render source code completely unreadable.  

**[More details](https://confluence.devfactory.com/x/QAPvEw)**
          
## The possible solution in this case

```JAVA
public void givenLocaleWhenGetLocaleThenReturnUtilLocale() {
                      ^

```
 

<a id="d15fa4d0-48ef-11e8-90f2-2f455003fed5" href="none" codereview-uuid-reserved-tag />